service/src/main/resources/api/service_openapi.yaml

```diff
@@ -598,6 +598,32 @@ components:
         organization:
           $ref: '#/components/schemas/Organization'
 
+    ChangeLogEntry:
```

Comment:
This doesn't seem to be used in an API controllers, is this here so we can get an auto generated model for the DAO? If so, an internal representation should be used instead which can then be marshaled to API models down the line (something like [this](https://github.com/DataBiosphere/terra-billing-profile-manager/blob/main/service/src/main/java/bio/terra/profile/service/profile/model/BillingProfile.java#L14).

Reply:
I added an internal model. 
I left this here, since we intend on adding the endpoints imminently.